src/api/private/auth/auth.controller.ts

```diff
@@ -70,14 +70,20 @@ export class AuthController {
     @Body() changePasswordDto: UpdatePasswordDto,
   ): Promise<void> {
     try {
+      await this.identityService.loginWithLocalIdentity(
```

Comment:
I think we should rename this function if we want to use it here, maybe something like `checkLocalPassword`?

Reply:
Done